hub/src/main/java/ch/prevo/open/hub/repository/NotificationDTO.java

```diff
@@ -0,0 +1,115 @@
+package ch.prevo.open.hub.repository;
+
+import ch.prevo.open.encrypted.model.MatchNotification;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.time.LocalDate;
+
+@Entity
+public class NotificationDTO {
```

Comment:
For me it is very confusing that the name ending DTO is used for an entity. I have only seen DTOs to be used as objects which are used by a REST API. Is that a common standard?

Reply:
It is common, I have seen it in several occasions. In general DTOs are used to carry data between remote processes. The original intent was to use it in remote interfaces, e.g. a REST interface. One could argue that exchanging data between an application and the DB is also communication between two remote processes, therefore speaking about DTOs at this point makes sense.

Technically a DTO has two responsibilities: to aggregate data that can be send in a single call and to take care of serialization. Entities do not really aggregate data, but they do take care of serialization. Guess one could argue both ways.